cmake/Platform/Common/Install_common.cmake

```diff
@@ -182,6 +194,23 @@ function(ly_setup_target OUTPUT_CONFIGURED_TARGET ALIAS_TARGET_NAME absolute_tar
     list(REMOVE_DUPLICATES INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER)
     string(REPLACE ";" "\n" INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER "${INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER}")
 
+    # If the target is an executable/application, add a custom target so we can debug the target in project-centric workflow
+    if(should_create_helper)
+        string(REPLACE ".Imported" "" RUN_TARGET_NAME ${NAME_PLACEHOLDER})
```

Comment:
Instead of removing the `.Imported` from the NAME_PLACEHOLDER, we can just have a variable that sets the original placeholder before appending the `.Imported` text.

Reply:
NAME_PLACEHOLDER comes from a conditional, I would have to fill it up for both cases. If you feel that makes it easier to read, I can change it

Reply:
I feel weakly about this.
You can leave it as is.